### DISCUSSION TITLE
How to create sub Navigation ?

### DISCUSSION THREAD

**Discussion by hblab-hautp** [Budibase/budibase#3872] [Help]
Hi team, I've just receive a design from my customer, according to which I must create sub navigation to group same features like this
![Screenshot from 2021-12-27 11-06-38](https://user-images.githubusercontent.com/95276487/147433333-70d58391-0805-415d-85df-57aea70ac53f.png)
I have tried to research in repo's discussions but i have no anwser.
So, have any way to create a sub navigation for my case ?  
Thank you !

**Comment by mjashanks** [comment]
There is no in-built way to do this. However, it might be possible to build it yourself using containers.

1. Go to `Layouts` and choose a layout
2. Set `Navigation` to `none`
3. Add a Container "Root Container" below
4. Put the `Screen Slot` inside the root container
5. Set the Root Container's `Direction` to `Row`
6. Put a container inside your root container, to act as your nav bar
7. Build out your nav bar from here

You may be able to use conditional UI to make the currently selected item display differently, based on URL.

My Structure looks like:

![image](https://user-images.githubusercontent.com/3524181/147538581-b88e8d75-7203-4bf0-b245-ddf80e48192a.png)
